<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Similar question finder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTextEdit" name="input_textEdit">
      <property name="placeholderText">
       <string>Input your question here</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QListWidget" name="out_listWidget"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="find_Button">
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="listWidget">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
